### DISCUSSION TITLE
[1146] Table 'acore_world.charsections_dbc' doesn't exist

### DISCUSSION THREAD

**Discussion by vnasing** [mod-playerbots/mod-playerbots#465] [Q&A]
Initialize Data Stores...
[1146] Table 'acore_world.charsections_dbc' doesn't exist
Your database structure is not up to date. Please make sure you've executed all queries in the sql/updates folders.

#----------------------------------------------------------------------#

>> ABORTED

# Location 'D:\azerothcore\azerothcore-wotlk-Playerbot\src\server\database\Database\MySQLConnection.cpp:640'
# Function 'MySQLConnection::_HandleMySQLErrno'

#----------------------------------------------------------------------#

Compiled in Windows environment, running liyunfan1223 playerbots version, this error occurs. Azerothcore official does not have this error, please provide guidance.

**Comment by noisiver** [comment]
The characters and world database files aren't being picked up by the auto updater. You'll have to run them yourself or add them to the corresponding data/sql/custom folder in the core.

They should be picked up though, just saying.

**Reply by umass-daniel-nelson** [reply]
I'm having the same issue, I'm trying to figure out exactly what steps to take, I've tried copying the sql files into data/sql/custom folder and that didn't work. I've tried adding volume mounts to the modules folder in the docker-compose-override.yml file and so far that hasn't worked either.
